### ISSUE TITLE
Allow typical waitgroup patterns

### ISSUE THREAD

**Issue by andig** [tommy-muehle/go-mnd#4]
Got the following error:

```
server.go:203:10: mnd: Magic number: 1, in <argument> detected (gomnd)
                wg.Add(1)
```

Imho that's too picky. Should consider if a solitary `1`- in context of waitgroups or maybe even in any context- should not be considered magic.

**Comment by tommy-muehle** [comment]
Hi @andig, 
yes, that's true. I'll add this as an exclude rule.

**Comment by tommy-muehle** [comment]
Fixed with [v2.0.0](https://github.com/tommy-muehle/go-mnd/releases/tag/v2.0.0).
I decided to allow in general the number 1 because there are more false-positives as real cases.
